test/circle/install.sh

```diff
@@ -22,6 +22,12 @@ sudo mv linux-amd64/helm /usr/local/bin
 rm -f helm-${HELM_LATEST_VERSION}-linux-amd64.tar.gz
 rm -rf linux-amd64
 
+# Setup Helm so that it will work with helm dep commands. Only the client
+# needs to be setup. In addition, the incubator repo needs to be
+# available for charts that depend on it.
+helm init -c
+helm repo add incubator https://kubernetes-charts-incubator.storage.googleapis.com/
```

Comment:
Why is this line required? Shouldnt requiements.yaml/.lock just specify an absolute URL? What about stable?

Reply:
You have to add a repo before you can refer to it. Without adding the incubator repo you'd get errors like...

```sh
$ helm dep up
Error: no repository definition for https://kubernetes-charts-incubator.storage.googleapis.com/. Try 'helm repo add'
```

The stable repo is automatically added by default when `helm init` is run.

Reply:
Ok I see. Change looks good to me